Core/Core/Credentials/StreamWrapper.cs

```diff
@@ -319,13 +321,14 @@ public async Task<Account> GetAccount()
         _account = acc;
         return acc;
       }
-      catch (Exception e)
+      catch (Exception ex) when (!ex.IsFatal())
       {
-        err = e;
+        err.Add(new SpeckleException($"Account {acc} failed to auth stream wrapper", ex));
       }
     }
 
-    throw err;
+    AggregateException inner = new(null, err);
```

Comment:
I noticed here, we would only include the exception from the last account that failed. I decided it might be useful to have the exceptions from all of them.

Reply:
Oh this is great